ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.13.yaml

```diff
@@ -232,6 +232,23 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-hypershift-worker-node
     workflow: cucushift-installer-rehearse-aws-ipi-ovn-hypershift-guest
+- as: aws-ipi-ovn-hypershift-worker-node-inplace-p1-f14
```

Comment:
Could you help to change `aws-ipi-ovn-hypershift-worker-node-p1-f4` to `aws-ipi-ovn-hypershift-worker-node-replace-p1-f4`

Reply:
We should change the test frequency to f7 for worker-node upgrade